### ISSUE TITLE
pre-trained weight on a public drive

### ISSUE THREAD

**Issue by zabir-nabil** [GreatJiweix/Deep-multi-metric-learning-for-text-independent-speaker-verification#3]
Can you kindly share the pretrained weight in a public drive so it can be downloaded?
